app/routes/questionnaire.py

```diff
@@ -76,7 +80,7 @@ def before_post_submission_request():
     )
 
 
-@questionnaire_blueprint.route("/", methods=["GET"])
+@questionnaire_blueprint.route("/", methods=["GET", "POST"])
 @login_required
```

Comment:
There are still a number of `login_required` annotations that are no longer necessary now that the two blueprints in this file have it.